Drivers/Inc/MotorController.h

```diff
@@ -50,7 +50,7 @@ void MotorController_Init(float busCurrentFractionalSetPoint);
 
 /**
  * @brief   Sends MOTOR DRIVE command on CAN3
- * @param   newVelocity desired motor velocity setpoint in rpm
+ * @param   newVelocity desired motor velocity setpoint in m/s
```

Comment:
Did this actually change? The motor controller expects the new velocity in RPM, so the that's what the old driver took as an argument. I wouldn't change the documentation until the change is actually made.